var/spack/repos/builtin/packages/gmake/package.py

```diff
@@ -6,7 +6,6 @@
 import os
 import re
 
-from spack.build_environment import MakeExecutable, determine_number_of_jobs
```

Comment:
Where are we getting `MakeExecutable` from?

Reply:
There are other similar changes below.

Reply:
lib/spack/spack/package.py

Reply:
I see, it was _already_ there - so not visible in the diff :slightly_smiling_face: